### DISCUSSION TITLE
The Dockerfile that works

### DISCUSSION THREAD

**Discussion by AmazeQiu** [ScrapeGraphAI/Scrapegraph-ai#539] [Show and tell]
I used the Dockerfile listed in the project to build an image which just didn't work for runing the demo code in README.md.
It still needs more installation about scrapegraphai[burr] and Playwright.

The following Dockerfile works well for me:

``` docker
FROM python:3.11-slim

RUN apt-get update && apt-get upgrade -y

RUN pip install scrapegraphai
RUN pip install scrapegraphai[burr]

RUN python3 -m playwright install-deps
RUN python3 -m playwright install
```

**Comment by VinciGit00** [comment]
thank you
